generator/src/main/java/com/scottlogic/deg/generator/GenerateExecute.java

```diff
@@ -40,7 +40,7 @@ public GenerateExecute(GenerationConfig config,
     @Override
     public void run() {
 
-        ValidationResult validationResult = validator.validateCommandLine(config);
```

Comment:
Genuine question - I thought we were avoiding injecting the config into the validator because it shouldn't depend on the configuration, but have it passed in, is this not the case?

Reply:
Ah, I did not know that.

Reply:
@sl-slaing is this the case?

Reply:
Preferably we would pass it around as an argument. There are a couple of places where it's passed as a dependency which are difficult to change

Reply:
changed config back to a parameter